common/config/.golangci.yml

```diff
@@ -32,15 +32,12 @@ linters:
     - goimports
     - govet
     # linters default enabled by golangci-lint .
-    - deadcode
```

Comment:
why remove these check

Reply:
it's deprecated, replace by `unused`.
same as project `istio`

Reply:
how about the below two

Reply:
```
WARN [runner] The linter 'varcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter.  Replaced by unused. 
WARN [runner] The linter 'deadcode' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter.  Replaced by unused. 
WARN [runner] The linter 'structcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter.  Replaced by unused. 
```

Reply:
got it